src/UITests/AIWinSession.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Axe.Windows.Automation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+
+namespace UITests
+{
+    [TestClass]
+    public class AIWinSession
+    {
+        protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+
+        protected static WindowsDriver<WindowsElement> session;
+        protected static int processId;
+
+        public static void Setup(TestContext context)
+        {
+            // Launch a new instance of application
+            if (session == null)
+            {
+                // Create a new session to launch application
+                DesiredCapabilities appCapabilities = new DesiredCapabilities();
+
+                var currentDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                var exeFiles = Directory.GetParent(currentDir).Parent.Parent.GetFiles("AccessibilityInsights.exe", SearchOption.AllDirectories);
+
+                if (exeFiles.Length > 0)
+                {
+                    appCapabilities.SetCapability("app", exeFiles[0].FullName);
```

Comment:
Does this just happen to work because the executable we're looking for is first alphabetically?

Reply:
Yes, this is one of the open questions mentioned in the PR description. We need to find the path to the appropriate AccessibilityInsights.exe, e.g. inside the bin folder and debug and/or release. I'd like to find something better than this searching approach. Once we have improvements / resolve the open questions I'll refactor a bit and remove the do-not-merge tag.